concept.tex

```diff
@@ -332,9 +332,14 @@ \subsubsection{Science Applications}\label{sec:science-applications}
 \begin{itemize}
 \item \TODO{Add examples from Ifremer}
 \item \TODO{Add examples from UiO}
-\item Reproduciblity of result computed
-  with the ab-initio software Octopus
-  (\href{https://octopus-code.org}{https://octopus-code.org}) which provides a
+\item Reproducibility of data processing pipelines, at the example of a
+  Biophysics application\footnote{https://gitlab.mpcdf.mpg.de/MPIBP-Hummer/glycoshield-md}:
+  While the compute requirements are moderate and satisfied by Binder in the Cloud,
+  the underlying software stack is highly complex (it requires the Gromacs simulation
+  code (\href{https://www.gromacs.org}{https://www.gromacs.org}), among others)
+  and poses a particular challenge for reproducibility.
```

Comment:
Thanks, Klaus. Are the software requirements here something that the current version of repo2docker cannot deal with (because of the complexity of the software requirements)? Just to understand if this touches any of the new features.

Reply:
And: is there an image we could use? (For figure 1.2.3)?

Reply:
Technical background: From the environment.yml file, the image with the software stack can be built by repo2docker right now but maybe (likely) not in 1 or 2 years due to changing versions on the Internet. It uses conda-forge and Bioconda.  So I created a huge freezed and versioned base Docker image containing all the software, and Binder only installs our Glycoshield package and data on top (which also speeds up the build process a lot).

Reply:
I asked the main author of the package for his consent and an image ...

Reply:
The right side of this one would fit nicely https://gitlab.mpcdf.mpg.de/MPIBP-Hummer/glycoshield-md/-/raw/refactoring/TUTORIAL/IMG/tut1.svg.png .

Reply:
I'll merge this - I think it is good to have even without the figure. If you get the figure, let's add it later.